TransTest.hs

```diff
@@ -52,7 +52,7 @@ testHandshakeConcept = do
     assertEq (sort (arcs handshakeConcept)) (sort expected)
   where
     handshakeConcept = inputs[a] <> outputs [b] <> handshake00 a b
-    expected = [AndCausality (rise a) (rise b), AndCausality (rise b) (fall a), AndCausality (fall a) (fall b), AndCausality (fall b) (rise a)]
+    expected = [Causality [rise a] (rise b), Causality [rise b] (fall a), Causality [fall a] (fall b), Causality [fall b] (rise a)]
```

Comment:
Let's also fit all lines to 80 characters if possible, so that it's easy to read in GitHub without scrolling to the right.